tools/project_manager/show_word_context.js

```diff
@@ -27,7 +30,7 @@ window.addEventListener("DOMContentLoaded", function () {
     mainSplit.reLayout();
 
     function setSplitLink() {
-        switchLink.textContent = splitVertical ? proofIntData.strings.layoutHorizontal : proofIntData.strings.layoutVertical;
+        switchLink.textContent = splitVertical ? translate.gettext("Change to vertical layout") : translate.gettext("Change to horizontal layout");
```

Comment:
This is backwards -- if the split is vertical, shouldn't the link be "switch to horizontal"?

Reply:
Good catch! This was a lot of copy/paste and  got that one backwards.